crates/bevy_reflect/src/func/into_function.rs

```diff
@@ -0,0 +1,298 @@
+use crate::func::function::DynamicFunction;
+use bevy_utils::all_tuples;
+
+/// A trait for types that can be converted into a [`DynamicFunction`].
+///
+/// # Blanket Implementation
+///
+/// This trait has a blanket implementation that covers many functions, closures, and methods.
+/// And though it works for many cases, it does have some limitations.
+///
+/// ## Arguments
+///
+/// Firstly, the function signature may only have up to 15 arguments
```

Comment:
A link whining about the lack of variadic generics would be useful context here.

Reply:
Please Rust! 😭

Reply:
@alice-i-cecile any good link you had in mind for that?

Reply:
https://poignardazur.github.io/2024/05/25/report-on-rustnl-variadics/ is nice for background, and there doesn't seem to be a proper issue on Rust to link to, only abandoned RFCs 🙃